.gitlab-ci.yml

```diff
@@ -1250,38 +1289,64 @@ deploy_process_and_sysprobe:
 # Docker releases
 #
 
-tag_release:
+tag_release_6:
   <<: *docker_tag_job_definition
   <<: *run_when_triggered_on_tag
   stage: deploy
   when: manual
   variables:
     <<: *docker_hub_variables
   script:
-    - inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG} datadog/agent:${CI_COMMIT_TAG}
-    - inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG}-jmx datadog/agent:${CI_COMMIT_TAG}-jmx
+    - inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG}-py2 datadog/agent:${CI_COMMIT_TAG}
+    - inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG}-py2-jmx datadog/agent:${CI_COMMIT_TAG}-jmx
     - inv -e docker.publish --signed-push ${SRC_DSD}:${SRC_TAG} datadog/dogstatsd:${CI_COMMIT_TAG}
 
-latest_release:
+tag_release_7:
   <<: *docker_tag_job_definition
   <<: *run_when_triggered_on_tag
   stage: deploy
   when: manual
   variables:
     <<: *docker_hub_variables
   script:
-    - inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG} datadog/agent:latest
-    - inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG}-jmx datadog/agent:latest-jmx
+    - inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG}-py3 datadog/agent:${CI_COMMIT_TAG}
+    - inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG}-py3-jmx datadog/agent:${CI_COMMIT_TAG}-jmx
+
+latest_release_6:
+  <<: *docker_tag_job_definition
+  <<: *run_when_triggered_on_tag
+  stage: deploy
+  when: manual
+  variables:
+    <<: *docker_hub_variables
+  script:
+    - inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG}-py2 datadog/agent:latest
+    - inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG}-py2-jmx datadog/agent:latest-jmx
+    # TODO: remove previous lines and uncomment the following when agent 7 is released
+    #- inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG}-py2 datadog/agent:latest-py2
+    #- inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG}-py2-jmx datadog/agent:latest-py2-jmx
     - inv -e docker.publish --signed-push ${SRC_DSD}:${SRC_TAG} datadog/dogstatsd:latest
 
+latest_release_7:
+  <<: *docker_tag_job_definition
+  <<: *run_when_triggered_on_tag
+  stage: deploy
+  when: manual
+  variables:
+    <<: *docker_hub_variables
+  script:
+    - inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG}-py3 datadog/agent:latest
```

Comment:
Same here: agent7 is not ready yet.

Reply:
I think this is different from your comment above: the other job we won't trigger it by mistake, overriding 6.x tags. This one is risky because if someone clicks `latest agent 6` and `latest agent 7`, we're going to push ppl to agent 7 like it or not.

Here I'm advocating for commenting out this line, and the next, and adding the following:
```console
inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG}-py3 datadog/agent:latest-py3
inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG}-py3-jmx datadog/agent:latest-py3-jmx
```

This way we can already push these when we want without risking overriding the py2 ones (which prime over agent 7 for now)

Reply:
Probably more wise indeed 👍